lib/applicationinsights.json

```diff
@@ -1,4 +1,5 @@
 {
+  "connectionString": "${file:/mnt/secrets/civil-sdt/civil-sdt-appinsights-connection-string}",
   "role": {
     "name": "civil-sdt"
```

Comment:
Could this be "Civil SDT" as in name description or is this linked elsewhere?

Reply:
The [documentation for the role name](https://learn.microsoft.com/en-us/azure/azure-monitor/app/java-standalone-config#cloud-role-name) just says it's used to label the component on the Application Insights application map.  The application map appears to be that displayed when the "Application map" link is selected for an instance of Application Insights, e.g. [rd-aat Application Insights Application map](https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/1c4f0704-a29e-403d-b719-b90c34ef14c9/resourceGroups/rd-aat/providers/Microsoft.Insights/components/rd-aat/applicationMap)

Some projects set it to the same application name used in application.yaml (e.g. rd-professional-api [applicationinsights.json](https://github.com/hmcts/rd-professional-api/blob/master/lib/applicationinsights.json#L4) and [application.yaml](https://github.com/hmcts/rd-professional-api/blob/master/src/main/resources/application.yaml#L25)) and some set it differently (e.g. cmc-claim-store [applicationinsights.json](https://github.com/hmcts/cmc-claim-store/blob/master/lib/applicationinsights.json#L4) and [application.yaml](https://github.com/hmcts/cmc-claim-store/blob/master/src/main/resources/application.yml#L24)).

As the value just appears to be used for a label it doesn't look like it needs to match a name elsewhere in the code.